### ISSUE TITLE
dataset of training Critic

### ISSUE THREAD

**Issue by leeqiaogithub** [jantic/DeOldify#253]
I am confuse of dataset of training Critic.
In the code: data_crit = get_crit_data([name_gen, 'test'], bs=bs, sz=sz).It seems that only images from folder 'test' and 'name_gen' are used.

**Comment by leeqiaogithub** [comment]
I want to know why we do not use images in train and val folder.Is it because of the large amount of data too large?

**Comment by jantic** [comment]
@leeqiaogithub You only need a set of "real" images and generated images to train the critic.  You can go about this many ways but the way I went about it was to take advantage of the test set in imagenet that I otherwise wasn't using.  It was the right size for my purposes and I wanted something to train on that was separate from the GAN training portion for better generalization.

**Comment by leeqiaogithub** [comment]
> I otherwise wasn't using
'''
path = Path('data/imagenet/ILSVRC/Data/CLS-LOC')
def get_crit_data(classes, bs, sz, pct=1.0):
    src = ImageList.from_folder(path, include=classes, recurse=True)...
'''
according to the source code of ImageList.from_folder(), it seems all images in CLS-LOS(including train/val/test) is used.

**Comment by jantic** [comment]
@leeqiaogithub it's filtering the folders with the classes variable.  So what the critic winds up getting is two classes to classify, as it's pretrained as a binary classifier.  It's a bit hacky now that I look at it again after almost two years.

 data_crit = get_crit_data([name_gen, 'test'], bs=bs, sz=sz).
